docs/commandline.md

```diff
@@ -21,23 +21,24 @@ rules of [API doc generation](apidocs.html). The input accepted is described in
 
 The command line options accepted are:
 
-| Command Line Option                 |  Description    |
-|:-----------------------|:-----------------------------------------|
-| `--input`     |   Input directory of content (default: `docs`) |
-| `--projects`     |   Project files to build API docs for outputs, defaults to all packable projects |
-| `--output`         |           Output Directory (default 'output' for 'build' and 'tmp/watch' for 'watch') |
-| `--noapidocs`       |           Disable generation of API docs |
-| `--eval`             |         Evaluate F# fragments in scripts |
-| `--saveimages`        |        Save images referenced in docs |
-| `--nolinenumbers`       |      Don't add line numbers, default is to add line number. |
-| `--parameters`            |    Additional substitution parameters for templates |
-| `--nonpublic`           |      The tool will also generate documentation for non-public members |
-| `--nodefaultcontent`      |    Do not copy default content styles, javascript or use default templates |
-| `--clean`                 |    Clean the output directory |
-| `--help`                  |    Display this help screen |
-| `--version`               |    Display version information |
-| `--properties`           |    Provide properties to dotnet msbuild, e.g. --properties Configuration=Release Version=3.4 |
-| `--strict`               |    Fail if docs are missing or can't be generated |
+| Command Line Option  | Description                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Sorry for updating the entire table, the IDE aligned everything automatically 😅.